webapp-django/crashstats/crashstats/views.py

```diff
@@ -2019,7 +2019,8 @@ def format_float(number):
             'product': r['product_name'],
             'version': r['version_string'],
             'percentage': format_float(r['percentage']),
-            'numberOfCrashes': r['report_count']})
+            'numberOfCrashes': r['report_count'],
+            'rank': r.get('currentRank', '')})
```

Comment:
`currentRank` is a TCBS thing (Top Crashers by Signature). You're trying to use data from the signature summary.